### DISCUSSION TITLE
Enable quickly the app on MacOS without executing the command in README

### DISCUSSION THREAD

**Discussion by jwallet** [tareqimbasher/NetPad#90] [General]
Hey,

> TLDR: Right-click on "NetPad.app" and select Open (contextual menu), then in the warning modal select "Open" again. 

just to let you know that on MacOS if you download a new build, like always the system will tell you that

![image](https://github.com/tareqimbasher/NetPad/assets/23088305/132f81e8-414c-433e-a0d6-9170bcb1e468)

> MacOS cannot verify that this app is free from malware

and that it cannot be opened, and you have to move it to trash.

Well if instead of double clicking on it to open it, you just `right-click` and select `open`, the same modal will be displayed but this time with the option to open it by ignoring the security warning

![image](https://github.com/tareqimbasher/NetPad/assets/23088305/9cb08cc7-e8bc-42d5-a4d8-f3eaeeb1f0de)

TIP: If you move "NetPad.app" directly into your "Applications" folder, MacOS will prompt the second modal instead and will remember that you trusted this app.

**Comment by tareqimbasher** [comment]
Great tip! Thank you
